drivers/include/periph/adc.h

```diff
@@ -104,10 +108,11 @@ typedef enum {
  *
  * The ADC line is initialized in synchronous, blocking mode.
  *
- * @param[in] line          line to initialize
+ * @param[in]   line        line to initialize
  *
- * @return                  0 on success
- * @return                  -1 on invalid ADC line
+ * @retval      0           success
+ * @retval      -1          on invalid ADC line
+ * @retval      <-1         on internal errors
```

Comment:
```suggestion
 * @retval     < -1         on internal errors
```

Minor readability nit

Reply:
I agree, but this doesn't get parsed correctly by Doxygen anymore. (It will tread "<" as value, and "-1 on internal errors" as description.)

What does work (at least on the current Doxygen release I'm using locally) is:

```
 * @retval      "< -1"      on internal errors
```

Should I go for that instead, or keep `<-1` without space?

Reply:
If its rendered correctly in the Doxygen just leave it as is.